tests/AppTest.php

```diff
@@ -6,16 +6,19 @@
 
 use atk4\core\AtkPhpunit;
 use atk4\ui\App;
+use atk4\ui\Layout\Admin;
 use atk4\ui\Template;
 
 class AppTest extends AtkPhpunit\TestCase
 {
     protected function getApp()
     {
-        return new App([
-            'catch_exceptions' => false,
-            'always_run' => false,
-        ]);
+        return new App(
```

Comment:
no :)

Reply:
?

Reply:
original CS is ok and better - we do not want code to be longer than needed when readability remains

Reply:
if you correct this, we can merge the phpdoc change

Reply:
Even php CS fixer does not complain about this. But if its so important to you, I will.